### ISSUE TITLE
OTLP HTTP exporter for Metrics 

### ISSUE THREAD

**Issue by lalitb** [open-telemetry/opentelemetry-cpp#1371]
Before opening a feature request against this repo, consider whether the feature should/could be implemented in the [other OpenTelemetry client libraries](https://github.com/open-telemetry/). If so, please [open an issue on opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/issues/new) first.

**Is your feature request related to a problem?**
If so, provide a concise description of the problem.

**Describe the solution you'd like**
What do you want to happen instead? What is the expected behavior?

**Describe alternatives you've considered**
Which alternative solutions or features have you considered?

**Additional context**
Add any other context about the feature request here.

**Comment by owent** [comment]
I can help to implement OTLP http exporter if necessary.

**Comment by esigo** [comment]
> I can help to implement OTLP http exporter if necessary.

Hi @owent, thanks for your interest. We need to first upgrade Protos #1290, which is blocked by #1291. Would you be interested in taking over #1291?
Thanks 🙏

**Comment by esigo** [comment]
> > > I can help to implement OTLP http exporter if necessary.
> > 
> > Hi @owent, thanks for your interest. We need to first upgrade Protos #1290, which is blocked by #1291. Would you be interested in taking over #1291? Thanks 🙏
> 
> Sure.This can assign to me.
> How can I make changes to #1290 ? Just raise a PR to [esigo:upgrade-opentelemetry-proto-to-0.15.0](https://github.com/esigo/opentelemetry-cpp/tree/upgrade-opentelemetry-proto-to-0.15.0) ?

Thank you very much. You may just raise another PR. I'll close mine.

**Comment by lalitb** [comment]
> Sure.This can assign to me.

Thanks @owent for helping on this.

**Comment by owent** [comment]
OK.
